src/component/mxgraph/MarkerConfigurator.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2020 Bonitasoft S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { MxGraphFactoryService } from '../../service/MxGraphFactoryService';
+import { mxgraph } from 'ts-mxgraph';
+
+export enum MarkerConstant {
+  ARROW_DASH = 'dash',
```

Comment:
:question: should we prefix the value by something like `bpmn.` to avoid any clash with new type that could be added to the `mxGraph` lib in the future